plugins/nodejs.json

```diff
@@ -9,7 +9,7 @@
     "shell": {
         "init_hook": [
             "mkdir -p {{ .Virtenv }}/corepack-bin",
-            "corepack enable --install-directory \"{{ .Virtenv }}/corepack-bin/\""
+            "if command -v corepack &> /dev/null; then corepack enable --install-directory \"{{ .Virtenv }}/corepack-bin/\"; fi"
```

Comment:
nit you could do:

`command -v corepack &> /dev/null && corepack enable --install-directory \"{{ .Virtenv }}/corepack-bin/\"`

which is a bit shorter and more portable.